master/static/srv/update_aggregated_queued_time.sql

```diff
@@ -17,6 +17,7 @@ day_agg AS (
     FROM task_stats, const, allocations
     WHERE
         allocations.allocation_id = task_stats.allocation_id
+        AND task_stats.start_time != '0001-01-01 00:00:00+00:00'::TIMESTAMPTZ
```

Comment:
0001-01-01 00:00:00+00 is NULL https://github.com/uptrace/bun/discussions/285

Reply:
start time is non-null, i think that is why it is getting saved as the zero value for `var x time.Time`

Reply:
Yeah, so I guess when Bun sees `StartTime` is nil, it saves it as 0001-01-01 00:00:00+00. We require start_time timestamptz NOT NULL.